backend/middleware/debug_middleware.py

```
@@ -0,0 +1,133 @@
+                logger.logger.error("=" * 80)
+                logger.logger.error("🔍 DIAGNÓSTICO DETALHADO DO ERRO 405:")
+                logger.logger.error(f"   Método usado: {method}")
+                logger.logger.error(f"   Path: {path}")
+                logger.logger.error(f"   Métodos permitidos: {allowed_methods}")
+                logger.logger.error(f"   Headers do request: {json.dumps(headers, indent=2)}")
+                logger.logger.error(f"   Query params: {query_params}")
+                logger.logger.error("=" * 80)
+
+            return response
+
+        except Exception as exc:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)